src/module_2/module_2_EDA.md

```diff
@@ -0,0 +1,2197 @@
+      <th>count_people</th>
+      <th>count_adults</th>
+      <th>count_children</th>
+      <th>count_babies</th>
+      <th>count_pets</th>
+      <th>number_of_orders</th>
+      <th>variant_id_count</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>4033</th>
+      <td>9a4b53affbe91ca5fd0c97f6d86e8a59ae894f3c7a69a8...</td>
+      <td>Proposition</td>
+      <td>UKI</td>
+      <td>2021-11-16 09:46:26</td>
+      <td>2021-11-01 00:00:00</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>1</td>
+      <td>320</td>
+    </tr>
+    <tr>
+      <th>3796</th>
+      <td>9b5b3679033da9e1f3a4def18667e40a7763d0d3ec9d8f...</td>
+      <td>Proposition</td>
+      <td>UKK</td>
+      <td>2022-01-19 21:40:39</td>
+      <td>2022-01-01 00:00:00</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>3</td>
+      <td>110</td>
+    </tr>
+    <tr>
+      <th>3708</th>
+      <td>11c967ad95d1e3a8f9767612e2f084aad75832cf019f8a...</td>
+      <td>Proposition</td>
+      <td>UKD</td>
+      <td>2021-01-06 08:52:51</td>
+      <td>2021-01-01 00:00:00</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>5</td>
+      <td>90</td>
+    </tr>
+    <tr>
+      <th>2098</th>
+      <td>af86901cbe9be971840f20e73e8af9d733f0887163f331...</td>
+      <td>Proposition</td>
+      <td>None</td>
+      <td>2021-03-10 10:09:10</td>
+      <td>2021-03-01 00:00:00</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>3</td>
+      <td>88</td>
+    </tr>
+    <tr>
+      <th>3140</th>
+      <td>27c2c38327ba83867c6291ee6bd0c35548edd256ed51c9...</td>
+      <td>Proposition</td>
+      <td>UKL</td>
+      <td>2020-07-13 20:25:43</td>
+      <td>2020-07-01 00:00:00</td>
+      <td>2.0</td>
+      <td>2.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>9</td>
+      <td>85</td>
+    </tr>
+  </tbody>
+</table>
+</div>
+
+
+
+
+```python
+sns.heatmap(df_users_merged2[['number_of_orders', 'variant_id_count']].corr(), annot=True, cmap='coolwarm')
+plt.show()
+```
+
+
+    
+![png](module_2_EDA_files/module_2_EDA_36_0.png)
+    
+
+
+Hay una relación positiva entre estas dos variables, pero esta asociación no es lo suficientemente fuerte como para considerarla significativa.
+
+**Estudio de la correlacion entre count_people, count_adults, count_children, count_babies, count_pets y el número de pedidos**
+
+
+```python
+column_names = ['count_people', 'count_adults', 'count_children', 'count_babies', 'count_pets']
+
+for column in column_names:
+    df_users_merged2[column] = df_users_merged2[column].fillna(0).astype(int)
+
+sns.heatmap(df_users_merged2[['number_of_orders', 'count_people', 'count_adults', 'count_children', 'count_babies', 'count_pets']].corr(), annot=True, cmap='coolwarm')
```

Comment:
I find the colours very misleading. One normally uses blue and read for positive and negative correlations, but no for correlation intensity as you are doing